### PR TITLE
new addon for store

### PR DESCRIPTION
**Pull request by realTiim**
Addon LeXx got added by me

[screenshot](https://fuck.healthcare/Q1BVVD4Up5.png?key=h2oFRAe8wXBMwf)

### CODE REVIEW

File: websites/L/LeXx/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "alpha.lexx.app"
+    ],
+    "description": {
+      "de": "LeXx bietet einen Discord Bot mit vielen Funktionen und Anwendungsmöglichkeiten",
+      "en": "LeXx offers a discord bot with awesome features"
+    },
+    "service": "LeXx",
+    "version": "1.0.3",
+    "logo": "https://lexx.app/favicon/favicon-96x96.png",
+    "thumbnail": "https://lexx.app/favicon/banner.png",
```
Comment (Bas950) [inline websites/L/LeXx/dist/metadata.json:22]: upload to imgur

File: websites/L/LeXx/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      "status.lexx.app",
+      "dev.lexx.app",
+      "alpha.lexx.app"
+    ],
+    "description": {
+      "de": "LeXx bietet einen Discord Bot mit vielen Funktionen und Anwendungsmöglichkeiten",
+      "en": "LeXx offers a discord bot with awesome features"
+    },
+    "service": "LeXx",
+    "version": "1.0.3",
```
Comment (Bas950) [inline websites/L/LeXx/dist/metadata.json:20]: version 1.0.0 on release

**Review by Bas950 — CHANGES_REQUESTED** [review]
Dont upload js file, fix codacy errors.

File: websites/L/LeXx/dist/metadata.json (hunk 1 above)
Comment (realTiim) [reply websites/L/LeXx/dist/metadata.json]: did it

File: websites/L/LeXx/dist/metadata.json (hunk 2 above)
Comment (realTiim) [reply websites/L/LeXx/dist/metadata.json]: did it

File: websites/L/LeXx/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+      "name": "Tim",
+      "id": "277043432063631360"
+    },
+    "url": "lexx.app",
```
Comment (Bas950) [inline websites/L/LeXx/dist/metadata.json:6]: why did you remove the other urls?

**Review by Bas950 — CHANGES_REQUESTED** [review]
Dont upload js file, fix codacy errors.

File: websites/L/LeXx/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    "description": {
+      "de": "LeXx bietet einen Discord Bot mit vielen Funktionen und Anwendungsmöglichkeiten",
+      "en": "LeXx offers a Discord Bot with many functions and uses"
+    },
+    "service": "LeXx",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/1gAssGQ.png",
```
Comment (Bas950) [inline websites/L/LeXx/dist/metadata.json:13]: logo must be at least 512x512

File: websites/L/LeXx/dist/metadata.json (hunk 3 above)
Comment (realTiim) [reply websites/L/LeXx/dist/metadata.json:6]: cause i noticed that they will be displayed in the store and only the main should be there

File: websites/L/LeXx/presence.ts [hunk 5]
```diff
@@ -0,0 +1,80 @@
+const presence = new Presence({
+  clientId: "393887855274885121"
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/L/LeXx/presence.ts:5]: merge consts

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/L/LeXx/dist/metadata.json (hunk 4 above)
Comment (realTiim) [reply websites/L/LeXx/dist/metadata.json]: did it

File: websites/L/LeXx/presence.ts (hunk 5 above)
Comment (realTiim) [reply websites/L/LeXx/presence.ts]: did it

File: websites/L/LeXx/dist/metadata.json (hunk 3 above)
Comment (Bas950) [reply websites/L/LeXx/dist/metadata.json:6]: But now these wont work 
image

Maybe use regex then?

Comment (realTiim) [reply websites/L/LeXx/dist/metadata.json:6]: thanks for hint added it to metadata

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]
